crawlers/mooncrawl/mooncrawl/reports_crawler/cli.py

```diff
@@ -571,13 +579,143 @@ def generate_report_nft_dashboard_handler(args: argparse.Namespace):
+        bucket=MOONSTREAM_S3_PUBLIC_DATA_BUCKET,
+        key=f"orangedao/{query_name}/data.json",
+    )
+
+    query_name = "transfers_feed"
+
+    for range in ranges:
+        params = {
+            "time_range": range["time_range"],
+        }
+
+        generate_report(
+            client=client,
+            token=args.moonstream_token,
+            query_name=query_name,
+            params=params,
+            bucket_prefix=MOONSTREAM_S3_PUBLIC_DATA_BUCKET_PREFIX,
+            bucket=MOONSTREAM_S3_PUBLIC_DATA_BUCKET,
+            key=f"orangedao/{query_name}/{range['time_range'].replace(' ','_')}_data.json",
+        )
+
+    # all time feed
+
+    params = {
+        "time_range": f"{int(time.time())} seconds",
+    }
+
+    generate_report(
+        client=client,
+        token=args.moonstream_token,
+        query_name=query_name,
+        params=params,
+        bucket_prefix=MOONSTREAM_S3_PUBLIC_DATA_BUCKET_PREFIX,
+        bucket=MOONSTREAM_S3_PUBLIC_DATA_BUCKET,
+        key=f"orangedao/{query_name}/all_time_data.json",
+    )
+
+    limits = [10, 100]
+
+    query_name = "largest_distributors"
+
+    for limit in limits:
+        params = {
+            "limit": limit,
+        }
+
+        generate_report(
+            client=client,
+            token=args.moonstream_token,
+            query_name=query_name,
+            params=params,
+            bucket_prefix=MOONSTREAM_S3_PUBLIC_DATA_BUCKET_PREFIX,
+            bucket=MOONSTREAM_S3_PUBLIC_DATA_BUCKET,
+            key=f"orangedao/{query_name}/{limit}_data.json",
+        )
+
+    query_name = "largest_recipients"
+
+    for limit in limits:
+        params = {
+            "limit": limit,
+        }
+
+        generate_report(
+            client=client,
+            token=args.moonstream_token,
+            query_name=query_name,
+            params=params,
+            bucket_prefix=MOONSTREAM_S3_PUBLIC_DATA_BUCKET_PREFIX,
+            bucket=MOONSTREAM_S3_PUBLIC_DATA_BUCKET,
+            key=f"orangedao/{query_name}/{limit}_data.json",
+        )
+
+    logger.info("Done")
+
+
 def main():
     parser = argparse.ArgumentParser()
 
     parser.set_defaults(func=lambda _: parser.print_help())
 
     subparsers = parser.add_subparsers()
 
+    general_queries_parser = subparsers.add_parser("queries", help="Queries commands")
+
+    general_queries_subparsers = general_queries_parser.add_subparsers()
+
+    general_queries_parser.add_argument(
+        "--moonstream-token",
+        required=True,
+        type=str,
+    )
+
+    general_queries_subparsers.add_parser(
+        "list",
+        help="List all queries",
+        description="List all queries",
+    ).set_defaults(func=list_user_queries_handler)
+
+    init_tokenonomics_parser = general_queries_subparsers.add_parser(
+        "init-tokenonomics",
+        help="Create all predifind query",
+        description="Create all predifind query",
+    )
+
+    init_tokenonomics_parser.add_argument(
```

Comment:
This flag not used in service systemd files but required

Reply:
Not required to run as systemd